CHANGES.md

```diff
@@ -1,5 +1,25 @@
 # Changes
 
+## edge-21.11.3
+
+This edge releases fixes a bug in the policy controller that was causing
```

Comment:
It's not really a bug in the policy controller... I might phrase this as "fixes a compatibility issue that prevented the policy controller from starting in some Kubernetes distributions"